Accessibility/Accessibility/ViewControllers/HomeShopVC.swift

```diff
@@ -7,24 +7,48 @@
 //
 
 import UIKit
+import URBNSwiftyConvenience
 
-class HomeShopVC: UIViewController {
-
+class HomeShopVC: UITableViewController {
+    private var categories: [String] = ["Mens", "Womens", "Home", "Sale"]
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        navigationItem.title = "Shop"
+        tableView.register(HomeShopTableViewCell.self, forCellReuseIdentifier: "categoryCell")
+        //whiteOut()
     }
     
+    func whiteOut() {
+        tableView.separatorStyle = .none
+        navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor:UIColor.white]
+    }
+}
 
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+extension HomeShopVC {
```

Comment:
I like the fact you put these TableView Delegate and DataSource methods in an extension, I would combine all these calls into one extension and add a MARK here to indicate that's what's going on here

Reply:
updated